package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mergeapi/merge-accounting-node",
-  "version": "1.0.6",
+  "version": "1.0.7",
```

Comment:
can you also update the version at https://github.com/merge-api/merge-accounting-node ? I must have missed that as well in the last manual update